code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -52,29 +52,27 @@
 	stealth_active = 1
 	if(istype(src.loc, /mob/living/carbon/human))
 		var/mob/living/carbon/human/M = src.loc
-		spawn(0)
-			anim(M.loc,M,'icons/mob/mob.dmi',,"cloak",,M.dir)
+		addtimer(GLOBAL_PROC, "anim", 0, FALSE, M.loc, M, 'icons/mob/mob.dmi',
```

Comment:
I'd just like to note here, because it pisses me off, that anim() and the /atom/movable/overlay type are both abominations and someone should get rid of them.